app/services/affiliate-slots-targeting.json

```diff
@@ -5,9 +5,6 @@
       "fandom-com",
       "fandom-com2"
     ],
-    "wikiId": [
```

Comment:
Let's not commit that change to dev - I'll do that in separate branch that will be released only to sandbox.

Reply:
https://github.com/Wikia/mobile-wiki/pull/1808/commits/e22df60c37868bc14b44cd14bf11b55b50ff1ded